### PR TITLE
issues/12 Add SSL support

### PR DESCRIPTION
**Pull request by whiteluna**
https://github.com/acsylla/acsylla/issues/12

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Install Linux driver dependencies
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
-          sudo apt-get install libuv1 libuv1-dev
+          sudo DEBIAN_FRONTEND=noninteractive apt-get -y install libuv1 libuv1-dev default-jre
```
Comment (pfreixes) [inline .github/workflows/ci.yml:33]: what´s the reason for having to provide the  `DEBIAN_FRONTEND=noninteractive` and the `default-jre` ?

File: README.rst [hunk 2]
```diff
@@ -92,6 +92,28 @@ Example for use `Shard-Awareness <https://github.com/scylladb/cpp-driver/tree/ma
...
+        ssl_private_key = f.read()
+    with open('./certs/trusted.cert.pem') as f:
+        ssl_trusted_cert = f.read()
+
+    cluster = create_cluster(['localhost'],
+                             ssl_enabled=True,
+                             ssl_cert=ssl_cert,
+                             ssl_private_key=ssl_private_key,
+                             ssl_trusted_cert=ssl_trusted_cert,
+                             ssl_verify_flags=acsylla.SSLVerifyFlags.PEER_IDENTITY)
```
Comment (pfreixes) [inline README.rst:113]: awessome!

**Review by pfreixes — CHANGES_REQUESTED** [review]
Did we update the submodule? I can see changes commited.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (whiteluna) [reply .github/workflows/ci.yml:33]: @pfreixes default-jre need for java keytool command (used in tests for generate SSL certs for cassandra) DEBIAN_FRONTEND=noninteractive for install dependencies for default-jre which require interactive answers

**Pull request merged**